website/source/docs/builders/vmware-iso.html.md

```diff
@@ -28,6 +28,25 @@ scratch, booting it, installing an OS, provisioning software within the OS, then
 shutting it down. The result of the VMware builder is a directory containing all
 the files necessary to run the virtual machine.
 
+### VMware Player on Linux
+
+There are (as of 2016-Apr-18) two versions of VMware Player currently available for download:
+
+* [7.0](https://my.vmware.com/en/web/vmware/free#desktop_end_user_computing/vmware_player/7_0)
+* [12.0](https://my.vmware.com/en/web/vmware/free#desktop_end_user_computing/vmware_workstation_player/12_0)
```

Comment:
Hm.. I just re-read the caveats about newer operating systems so Player 12 is better. I don't know the details here, though. For instance, if I try to build a Windows 10 VM or an Ubuntu 16.04 VM, is player 7 not going to work _period_ or does it just not support drag-n-drop stuff (which doesn't matter)?

If newer operating systems _don't work period_ I think this is problematic but if it's just some more advanced features I don't care about that. Since this is official docs for packer I would like a path that _just works_ as much as possible.

Thanks again for taking the time to look into this and PR, this is unfortunately more complicated than it should be.